Makefile

```diff
@@ -298,3 +315,41 @@ clean: clean-release-manifest clean-bundle
 	@go clean -i $(PKGS)
 	@echo "Deleting image "$(OPERATOR_IMG)
 	@docker rmi -f $(OPERATOR_IMG) registry.access.redhat.com/ubi9-minimal:latest
+
+px-resource-gateway-proto-build-docker-proto:
+	docker build -t pure-artifactory.dev.purestorage.com/px-docker-dev-virtual/dgoel/px-resource-gateway-proto --network=host -f Dockerfile.proto .
```

Comment:
Use a name for the container like `px-resource-gateway-proto:dev` so that it is never pushed.